### ISSUE TITLE
Exception: Read gave wrong length

### ISSUE THREAD

**Issue by re-fox** [fireeye/capa#385]
### Description

This could be a viv bug.

```
Exception: Read gave wrong length at va: 0x140000ffc (wanted 8 got 4)
```

### Steps to Reproduce

1. `python2 scripts/show-features.py -F 0x140001020 40c0a313897b26e14e54683ceace4a161859f368b0114a2ca614deca3b2ca5da`
The following error is produced
```
  File "scripts/show-features.py", line 190, in <module>
    sys.exit(main())
  File "scripts/show-features.py", line 107, in main
    vw = capa.main.get_workspace(args.sample, args.format)
  File "/usr/lib/python2.7/site-packages/flare_capa-1.4.0-py2.7.egg/capa/main.py", line 267, in get_workspace
    vw = viv_utils.getWorkspace(path, should_save=should_save)
  File "/usr/lib/python2.7/site-packages/viv_utils-0.3.17-py2.7.egg/viv_utils/__init__.py", line 86, in getWorkspace
    vw.loadFromFile(fp)
  File "/usr/lib/python2.7/site-packages/vivisect-0.1.0-py2.7.egg/vivisect/__init__.py", line 2480, in loadFromFile
    fname = mod.parseFile(self, filename, baseaddr=baseaddr)
  File "/usr/lib/python2.7/site-packages/vivisect-0.1.0-py2.7.egg/vivisect/parsers/pe.py", line 36, in parseFile
    return loadPeIntoWorkspace(vw, pe, filename, baseaddr=baseaddr)
  File "/usr/lib/python2.7/site-packages/vivisect-0.1.0-py2.7.egg/vivisect/parsers/pe.py", line 333, in loadPeIntoWorkspace
    mapoffset = vw.readMemoryPtr(rva + baseaddr) - baseaddr
  File "/usr/lib/python2.7/site-packages/vivisect-0.1.0-py2.7.egg/envi/memory.py", line 189, in readMemoryPtr
    return self.readMemValue(va, self.imem_psize)
  File "/usr/lib/python2.7/site-packages/vivisect-0.1.0-py2.7.egg/envi/memory.py", line 178, in readMemValue
    raise Exception("Read gave wrong length at va: 0x%.8x (wanted %d got %d)" % (addr, size, len(bytes)))
Exception: Read gave wrong length at va: 0x140000ffc (wanted 8 got 4)
```

**Expected behavior:**
Executable to be processed

**Actual behavior:**
Exception is raised

### Versions
Master from git

### Additional Information
Sample is attached here  (password infected)
[mal.zip](https://github.com/fireeye/capa/files/5703693/mal.zip)

**Comment by williballenthin** [comment]
wondering/hoping if https://github.com/vivisect/vivisect/pull/338 is related.

**Comment by upside2** [comment]
I don't think so.  This sample executes without error in our production environment, which has #338.

**Comment by williballenthin** [comment]
![image](https://user-images.githubusercontent.com/156560/106166047-fde33e80-6148-11eb-8df7-f6ca46113bd2.png)

doesn't fix it. will dig into the issue.

**Comment by williballenthin** [comment]
triaged upstream. awaiting review and merge (or reject) of https://github.com/vivisect/vivisect/pull/347

if its rejected, then this is a situation of unsuccessfully trying to process a corrupt PE file. carrying on or rejecting such a binary are both valid strategies, imho.

**Comment by williballenthin** [comment]
fixed upstream. awaiting 0.2.0 of viv (maybe this week) and then we can close this issue.

**Comment by mr-tz** [comment]
Great, thanks for the fix! Let's release v1.5.0 after this.
